src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionExtensions.StreamAsChannelAsync.cs

```diff
@@ -8,63 +8,254 @@
 
 namespace Microsoft.AspNetCore.SignalR.Client
 {
+    /// <summary>
+    /// Extensions methods for <see cref="HubConnectionExtensions"/>.
+    /// </summary>
     public static partial class HubConnectionExtensions
     {
+        /// <summary>
+        /// Invokes a streaming hub method on the server using the specified method name and return type.
+        /// </summary>
+        /// <typeparam name="TResult">The return type of the streaming server method.</typeparam>
```

Comment:
The docs refer to `TResult` as the `ChannelReader<TResult>`. This can be confusing, we should change either the return generic or the doc comments

Reply:
`TResult` is the name of the generic parameter on the method and on `Task<TResult>`. Not sure what can be done.

IMO no one actually reads return types in VS intelli-sense anyway.

Reply:
These also end up on https://docs.microsoft.com/en-us/dotnet/api/index?view=aspnetcore-2.0 though